docs/src/content/installation/verification.md

```diff
@@ -0,0 +1,137 @@
+# Verifying that you are up and running
+
+## Prerequisites verification
+
```

Comment:
Maybe we should add running the cardano node is a prerequisite? I think it asks for the socket location.
@joseph-fajen